skxray/tests/test_core.py

```diff
@@ -538,6 +543,34 @@ def run_image_to_relative_xyi_repeatedly():
             print('{0} calls successful'.format(num_calls))
 
 
+def test_radial_integration():
+    calib_center = (50., 50.)
+    dist_s = 50  # (mm)
+    detector_size = (100, 100)
+    wavelength = 0.15  # (nm)
+    image = np.ones((detector_size))
+
+    # when the x axis is in radius
+    (bin_cen_r, ring_ave_r) = core.radial_integration(image, calib_center,
+                                                      num_bins=50)
+
+    # when the x axis is in two theta
+    (bin_cen_t, ring_ave_t) = core.radial_integration(image, calib_center,
+                                                      x_axis='two_theta',
+                                                      dist_sample=dist_s,
+                                                      num_bins=40)
+    # when the x axis is in Q space
+    (bin_cen_q, ring_ave_q) = core.radial_integration(image, calib_center,
+                                                      x_axis='q',
+                                                      wavelength=wavelength,
+                                                      dist_sample=dist_s,
+                                                      num_bins=20)
+
+    assert_array_equal(ring_ave_r, np.ones(49))
```

Comment:
This should return 50, I think there is on off by one bug in the code someplace...